.eslintrc.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
     'no-console': [0, 'log', 'error'],
     'no-var-requires': 0,
     '@typescript-eslint/explicit-function-return-type': 0,
+    '@typescript-eslint/no-explicit-any': 0,
```

Comment:
As mentioned, I'd rather see this as an ignore comment in the file instead of disabling the rule for the entire lib.